testsuite/settings.py

```diff
@@ -12,12 +12,14 @@
     }
 }
 
-CONSUL_URL = os.environ.get('TESTING_CONSUL_URL', 'http://localhost:8500')
-NOMAD_URL = os.environ.get('TESTING_NOMAD_URL', 'http://localhost:4646')
+CONSUL_URL = os.environ.get('TESTING_CONSUL_URL', 'http://10.66.60.1:8500')
+NOMAD_URL = os.environ.get('TESTING_NOMAD_URL', 'http://10.66.60.1:4646')
 NOMAD_JOB_PREFIX = f'testsuite-{random_code(8)}-'
 NOMAD_DEPLOYMENT_NAME = f"test {datetime.now().strftime('%H:%M:%S')}"
 
 VMCK_BACKEND = os.environ.get('TESTING_BACKEND', 'docker')
 
 _github_image = 'https://github.com/mgax/vmck-images/raw/master/bionic.qcow2'
 QEMU_IMAGE_URL = os.environ.get('TESTING_QEMU_IMAGE_URL', _github_image)
+
+SSH_USERNAME = os.environ.get('SSH_USERNAME', 'ubuntu')
```

Comment:
Let's default to `vagrant`. If it's missing from the docker mock vm, let's change it. We probably need a separate repo and CI pipeline for it. Let's call it `vmck/mock` on docker hub.